src/main/java/frc/team1816/robot/commands/GamepadArmCommand.java

```diff
@@ -0,0 +1,50 @@
+package frc.team1816.robot.commands;
+
+import com.edinarobotics.utils.gamepad.Gamepad;
+import edu.wpi.first.wpilibj.command.Command;
+import frc.team1816.robot.Components;
+import frc.team1816.robot.subsystems.Arm;
+
+public class GamepadArmCommand extends Command {
+    private Arm arm;
+    private double power = 0.5;
+    private Gamepad gamepad;
```

Comment:
I see that this is the way the old Zenith code worked too, but it is generally bad practice to refer to single instance variables such as gamepad by passing them around.  In this case we don't know if the gamepad was even initialized or even which gamepad it is.

It is safer to use ```Controls.getInstance().gamepadDriver``` anywhere that we need access to the gamepad.  So we can remove the gamepad member variable.

Copying @andrewfhou just so that we all see this comment.